_data/simple-icons.json

```diff
@@ -3060,6 +3060,11 @@
             "hex": "DE4F4F",
             "source": "http://logo.meteorapp.com/"
         },
+        {
+            "title": "micro:bit",
+            "hex": "0191DC",
```

Comment:
```suggestion
            "hex": "00ED00",
```

Reply:
Ah, feck! Somehow managed to copy & paste the wrong colour from that SVG file. The green in it is actually `#10d95e`. But, looking at the social media & apps, I'd agree with the `#00ed00` choice. Thanks, Eric.

Reply:
Would've been okay with both if it is sourced from the SVG, thanks for updating :+1: